src/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/roll.cpp

```diff
@@ -61,7 +61,7 @@ const auto testCaseNegativeUnorderedAxes4D = ::testing::Combine(
 );
 
 const auto testCaseRepeatingAxes5D = ::testing::Combine(
-    ::testing::Values(std::vector<size_t>{2, 16, 32, 32}), // Input shape
+    ::testing::Values(std::vector<size_t>{2, 16, 32, 7, 32}), // Input shape
```

Comment:
why did you changed cpu test?

Reply:
Because seems this was a typo in CPU test. The test should be for 5D, but the input shape is 4D, this is the same operation, so I added a quick fix.